vrops_exporter.py

```diff
@@ -30,13 +30,13 @@ def parse_params():
     if options.port:
         os.environ['PORT'] = options.port
 
-    if "PORT" not in os.environ and not options.port:
+    if "PORT" not in os.environ and not options.port or os.environ["PORT"] == ' ':
```

Comment:
is ` == '  '` and actual use case or what is the reason for this?

Reply:
I wonder if it could happen that the parameters are blank. I don't think so. I take this test case out.